### ISSUE TITLE
Summary Type for manage_status_definition

### ISSUE THREAD

**Issue by anthonyangel** [terraform-providers/terraform-provider-datadog#370]
Hi there,

Thank you for opening an issue. Please note that we try to keep the Terraform issue tracker reserved for bug reports and feature requests. For general usage questions, please see: https://www.terraform.io/community.html.

### Terraform Version
0.12.18

### Affected Resource(s)
Please list the resources as a list, for example:
- datadog dashboard

I would like to have support for the summary type field for the manage_status_definition widget. This is defaulting to type: monitor, but there is also type:group and type:combined as other available options. I can see that these are expressed in the json export of an existing dashboard

**Comment by Towerthousand** [comment]
@bkabrda I'd say this is not only a feature request but also a bug:

```
  widget {
    manage_status_definition {
      query            = "env:${var.env} AND team:${var.team} AND service:${var.service} AND tag:critical:true"
      display_format   = "countsAndList"
      sort             = "status,asc"
      color_preference = "background"
      hide_zero_counts = true
    }
    layout = {
      x      = 42
      y      = 10
      width  = 41
      height = 26
    }
  }
```

Though the query does return monitors, the generated widget shows up like this:

<img width="960" alt="Screenshot 2020-03-13 at 14 24 35" src="https://user-images.githubusercontent.com/3587309/76625539-c02cf580-6537-11ea-9b6f-0791865ed776.png">

So the provider won't generate a working resource at all.

If you go edit the resource manually and save with no changes, it works.

**Comment by Towerthousand** [comment]
Nevemind, I see it was already implemented. Thanks!

**Comment by bkabrda** [comment]
Yup, this was implemented and released in 2.7.0: https://github.com/terraform-providers/terraform-provider-datadog/blob/master/CHANGELOG.md#270-february-10-2020

Closing this issue, thanks everyone!
